u-root.go

```diff
@@ -332,3 +352,37 @@ func Main(l ulog.Logger, buildOpts *gbbgolang.BuildOpts) error {
 	}
 	return uroot.CreateInitramfs(l, opts)
 }
+
+func validateArg(arg string) bool {
+	if !checkPrefix(arg) {
+		if strings.Contains(arg, "*") {
+			paths, err := filepath.Glob(arg)
+			if err != nil {
+				return false
+			}
+			for _, path := range paths {
+				if !validateArg(path) {
+					return false
+				}
+			}
+		}
+		if info, err := os.Stat(arg); err != nil {
+			return false
+		} else if !info.IsDir() {
+			return false
+		}
+	}
+
+	return true
+}
+
+func checkPrefix(arg string) bool {
+	prefixes := []string{".", "/", "-", "cmds", "github.com/u-root/u-root"}
```

Comment:
It first checks the regular absolute and relative filepath prefixes and then probes for other provided filepaths further down.

If this import path is provided it will internally be converted to a filepaths and requires the `-uroot-source` flag to work and will give an error if the flag wasn't provided. This was just an attempt to break a bit less of the old functionality but I can also disallow this particular last import path as well.

Reply:
This seems fine to me as a convenient bridge to our new modules world?